sympy/matrices/tests/test_matrices.py

```diff
@@ -1878,6 +1878,8 @@ def test_errors():
     raises(ShapeError, lambda: Matrix([1, 2, 3]).dot(Matrix([1, 2])))
     raises(ShapeError, lambda: Matrix([1, 2]).dot([]))
     raises(TypeError, lambda: Matrix([1, 2]).dot('a'))
+    raises(SymPyDeprecationWarning, lambda: Matrix([[1, 2], [3, 4]]).dot(Matrix([[4, 3], [1, 2]])))
```

Comment:
@ylemkimon I had added a test for old behaviour.

Reply:
@siefkenj this is a test I had added for warning. Should I add another warning ?

Reply:
This tests tests that an error is raised, but it doesn't test that the output is what it should be.

Reply:
okay!
```
assert Matrix([[1,2],[3,4]]).dotMatrix([[1,3],[4,5]])) == [10, 19, 14, 28]
```
will this test work?

Reply:
That's closer, but the test will fail since it raises a warning. Lookup `warnings.catch_warning` to see how to catch the warning so it isn't treated as an error.